micro_http/src/response.rs

```diff
@@ -14,8 +14,12 @@ use headers::{Header, Headers, MediaType};
 #[allow(dead_code)]
 #[derive(Clone, Copy, PartialEq)]
 pub enum StatusCode {
-    /// 100, OK
+    /// 100, Continue
```

Comment:
You need to update the documentation of status code in `micro_http/src/lib.rs` and add the newly supported status codes.